### PR TITLE
Added Loop for Team and proper text in text.json

### PR DESCRIPTION
**Pull request by ghost**
Added loop to `Team` in `state/team` with the respective text in `text.json` to indicate that the user is indeed in the `Team` menu state. Upon exiting the `Team` loop there is no text printed, but the user will be notified that they are in the `Main Menu`.

### CODE REVIEW

File: states/team.py [hunk 1]
```diff
@@ -1,7 +1,35 @@
...
         super().__init__()
+        while self.option is not TeamOptions.EXIT:
+            print(TEXT["TEAM"]["ENTRY"])
+            self.option = self.check_input(TeamOptions)
+            self.choose_option()
+
+    def choose_option(self):
+        """Enters next game state based on player choice"""
+        if self.option == TeamOptions.TOSSPKMN:
+            Battle()
```
Comment (biggestcookie) [inline states/team.py]: We don't really want to be entering these states (`Battle()` and `Team()`) after each choice, even if they don't do anything yet. If you want to keep the if statements, just use `pass` to have a piece of code that doesn't do anything, as a placeholder for future code

File: states/team.py [hunk 2]
```diff
@@ -1,7 +1,35 @@
...
+        elif self.option == TeamOptions.REORDER:
+            Team()
+        elif self.option == TeamOptions.EXIT:
+            print(TEXT["TEAM"]["EXIT"])
```
Comment (biggestcookie) [inline states/team.py]: I don't think anything needs to be done on `TeamOptions.EXIT`. The game will already print to the player that they chose the choice 'exit'. By default, `choose_option` should end if the player did not choose one of `TeamOptions` that satisfies the other `if` conditions, so the `Team` State should exit to Menu by default. In other words, there's no need for this if statement, or the `EXIT` text on `TEAM`.

File: config/text.json [hunk 3]
```diff
@@ -9,5 +9,9 @@
     },
     "POKEMON": {
         "ADD": "joined your team!"
+    },
+    "TEAM": {
+        "ENTRY": "TEAM MENU",
```
Comment (biggestcookie) [inline config/text.json]: Let's call this 'MANAGE TEAM' instead, it's more descriptive for the user

**Comment by ghost** [comment]
Pushed code that should satisfy suggestions. Please review

File: config/text.json [hunk 4]
```diff
@@ -11,7 +11,7 @@
         "ADD": "joined your team!"
     },
     "TEAM": {
-        "ENTRY": "TEAM MENU",
+        "ENTRY": "MANAGE TEAM",
         "EXIT": " "
```
Comment (biggestcookie) [inline config/text.json]: No need for the "EXIT" text here, it's just an empty string anyways. If you wanted an empty line there are probably better, more reusable ways of doing that

File: states/team.py [hunk 5]
```diff
@@ -1,7 +1,33 @@
...
+
     def __init__(self):
         super().__init__()
+        while self.option is not TeamOptions.EXIT:
+            print(TEXT["TEAM"]["ENTRY"])
+            self.option = self.check_input(TeamOptions)
+            self.choose_option()
+
+    def choose_option(self):
+        """Enters next game state based on player choice"""
```
Comment (biggestcookie) [inline states/team.py]: When we choose an organization option, we're actually not going to create an entirely new game state just for the next step. We'll probably just enter a new function under the `class Team(State):`, and maybe offer a substate if there are more decisions to be made. At any rate whatever it is, just fix the docstring to describe that `choose_option` will perform the team action of their choice.

Comment (ghost) [reply states/team.py]: Changed to `Performs team action based on player choice`

File: config/text.json (hunk 4 above)
Comment (ghost) [reply config/text.json]: Deleted `"EXIT": " "`

**Pull request merged**